Breeding-Insight/bi-web
# [BI-1442] - Previous Banners Don't Vanish on New Import

# Description
**Story:** [BI-1442 Previous Import Error Banner Does Not Vanish on New Germplasm Import](https://breedinginsight.atlassian.net/browse/BI-1442)

Added call to remove all previous notifications at the start of user import, abort, and confirmation of import.

# Dependencies
bi-api/develop

# Testing
For Germplasm Import:

Upload file that causes banner error (ie missing GID)
- Upload new valid file and click import -  error banner should vanish

Click confirm of import without adding germplasm list name to get new error banner
- Click Abort - no list name error banner should vanish

Click confirm of import without adding germplasm list name to get new error banner
- Add list name and confirm - no list name error banner should vanish and successful import banner should appear

With successful import banner, import new germplasm
- Successful import banner from previous import should vanish

Also test similar functionality with Ontology import.

# Checklist:

- [X] I have run TAF: [_\<link to TAF run>_](https://github.com/Breeding-Insight/taf/actions/runs/2310592188)
